mlflow/server/js/src/experiment-tracking/components/ExperimentRunsTableMultiColumnView2.js

```diff
@@ -335,6 +335,10 @@ export class ExperimentRunsTableMultiColumnView2 extends React.Component {
     this.applyRowSelectionFromProps();
     this.handleColumnSizeRefit();
     this.handleLoadingOverlay();
+
+    window.addEventListener('resize', () => {
+      this.gridApi.sizeColumnsToFit();
```

Comment:
Resize event can be busy, what do you think about debouncing 100ms here?
```
window.addEventListener('resize', _.debounce(this.gridApi.sizeColumnsToFit, 100));
```

Reply:
Makes sense. I'll make the change